### PR TITLE
Link/unlink device calls

### PR DESCRIPTION
**Pull request by rubdos**
Calls intended to run on primary devices to allow and delete secondary devices. #59 contd.

### CODE REVIEW

**Comment by gferon** [comment]
Is this ready to merge? 🚀

**Comment by rubdos** [comment]
> Is this ready to merge? rocket

Not tested, I'll merge when it's tested. Looking good so far?

**Comment by rubdos** [comment]
@gferon Few possibly weird design choices, could you specifically have a look at my change in Actix' `push_service.rs`? Would like a second opinion there :-)

File: libsignal-service/src/provisioning.rs
Comment (gferon) [inline libsignal-service/src/provisioning.rs:412]: amazing 🥳

**Review by gferon — APPROVED** [review]
LGTM, awesome!

File: libsignal-service/src/provisioning.rs
Comment (rubdos) [reply libsignal-service/src/provisioning.rs:412]: :tada:

File: libsignal-service-actix/src/push_service.rs
Comment (rubdos) [reply libsignal-service-actix/src/push_service.rs:122]: Fixed every comment but this.  Move to #63, then I can merge ;-)

**Pull request merged**
